app/styles/onboarding-card.module.css

```diff
@@ -1,19 +1,95 @@
 .onboarding-card{
-    min-height: 60vh;
+    min-height: 55vh;
     min-width: 40vw;
     display: flex;
     flex-direction: column;
     justify-content: center;
     align-items: center;
-    border: 2px solid var(--color-lightgrey);
-    background-color: var(--color-cultured);
-    border-radius: 10px;
-    margin-top: 80px;
+    background-color: #FFF;
+    border-radius: 24px;
+    position: relative;
+    padding: 0px 16px;
+    box-shadow: 0px 4px 10px -2px rgba(16, 24, 40, 0.06), 0px 6px 15px -2px rgba(16, 24, 40, 0.10);
 }
-@media screen and (max-width: 625px) {
+
+.signup-details__rds-logo{
+    display: block;
+    position: absolute;
+    transform: translateX(-50%);
+    left: 50%;
+    top: -10%;
+}
+
+.logos-container{
+    padding: 25px;
+    display: flex;
+    flex-direction: row;
+    position: absolute;
+    transform: translateX(-50%);
+    left: 50%;
+    top: 3%;
+}
+.logo--github{
+    padding: 7px;
+}
+
+.logo--link{
+    position: absolute;
+    top: 35%;
+    left: 44%;
+    width: 20%;
+    height: 30%;
+}
+
+.logo--rds{
+    margin-left: 6%;
+    padding: 7px;
+}
+
+.logo__img--link{
+    width: 50px;
+    height: 50px;
+}
+
+.signup-page-heading{
+    color: var(--neutral-neutral-700, #272626);
+    text-align: center;
+    font-size: 22px;
+    font-style: normal;
+    font-weight: 700;
+    line-height: 24px; 
+}
+@media (max-width: 800px) {
     .onboarding-card{
-        min-height: 50vh;
-        max-width: 90vw;
-        margin: 40px auto;
+        min-width: 60vh;
+    }
+}
+
+
+@media (max-width: 625px) {
+    .onboarding-card{
+        min-height: 30vh;
+        min-width: 75vw;
+        padding: 56px 16px;
+        margin-top: -5rem;
+    }
+    .signup-details__rds-logo{
+        width: 80px;
+        height: 80px;
+    }
+
+    .logo__img--github{
+        height: 75px;
+        width: 80px;
```

Comment:
shouldn't the logos be having the same height and width? 🤔

Reply:
Done with changes can you please check latest commit